pkg/cloud/gcp/gcp.go

```diff
@@ -156,7 +159,7 @@ func (c *GcpCloud) SetupNetwork(ipAddress, networkName string) (*compute.Network
 
 // SetFirewallRule creates a new firewall rule in GCP
 func (c *GcpCloud) SetFirewallRule(ipAddress, firewallName, networkName string, ports []string) (*compute.Firewall, error) {
-	if strings.Contains(ipAddress, "/") {
+	if strings.Contains(ipAddress, "/") && ipAddress != "0.0.0.0/0" {
```

Comment:
do I read it correctly?
it should be `!strings.Contains(ipAddress, "/")` if I understand this code